templates/jobservice/jobservice-dpl.yaml

```diff
@@ -86,6 +86,13 @@ spec:
           - name: INTERNAL_TLS_TRUST_CA_PATH
             value: /etc/harbor/ssl/jobservice/ca.crt
           {{- end }}
+          {{- if .Values.registry.credentials.existingSecret }}
```

Comment:
should we delete line 89 - line 95 since there is no `.Values.registry.credentials.existingSecret`. 
If so, please remove those lines instead of adding `.Values.registry.credentials.existingSecret` because we would prefer to keeping the credentials of Harbor internal component as simple as possible.

Reply:
"existingSecret" has been added